Topics/Tech_Stacks/PostgreSQL_psycopg2.md

```diff
@@ -0,0 +1,74 @@
+- Windows: https://www.postgresql.org/download/windows/
+- MacOS: https://www.postgresql.org/download/macosx/
+- Linux: https://www.postgresql.org/download/linux/
+
+Once installed, you can create a database, relation, and tables with the PostgreSQL client called "pgAdmin". You can also perform queries on relations and tables through its query tool. More information on how to use pgAdmin can be found from this [link](https://www.pgadmin.org/docs/pgadmin4/6.21/index.html).
+
+If you are using MacOS, you can also run the command "brew install postgresql" in your terminal. Note that you also have to have Homebrew installed for this command to work. "brew install postgresql" allows the user to use PostgreSQL on their command line by simply initiating a PostgreSQL environment with the command "psql".
+
+For Windows users, it's better for the user to download PostgreSQL through the link provided above. If the user wants to utilize PostgreSQL on the command line, they must (after installation):
+1. Add the PostgreSQL bin directory path to the PATH environment variable.
+2. Run the command "psql -U username"
+
+To install psycopg2, you can use pip to install its functionality by using the command:
+"pip install psycopg2". After installing this in your desired directory, you can import psycopg2 into your python files and perform PostgreSQL queries very intuitively.
+
+### Basic Operations
+A very very quick and basic runover of PostgreSQL.
+
+To select data from a certain column in a table, you can use a query like:
+``` 
+    SELECT col_name1, col_name2 FROM table_name;
+```
+This gives you a smaller table that only contains the columns "col_name1" and "col_name2" from your selected table in your relation. 
+
+You can also merge two different tables by changing the FROM clause to contain 2 tables:
+```
+    SELECT col1_table1, col2_table2 FROM table1_name, table2_name;
+```
+This operation, first, performs a cross product between table1 and table2, where every row of one table is matched with every other row of the other table. Then it selects the data present in the new cross-products table from columns col1_table1 and col2_table2.
+
+In addition, you can perform filtering on tables:
+```
+SELECT col1_table1, col2_table2 FROM table1_name, table2_name WHERE col1_table1 > 1;
+```
+Here it only shows data from the previous table where the value of a row in the column "col1_table1" is greater than 1. This is the syntax for when the table and column names between table 1 and table 2 are unique.
+
+There are many other features of PostgreSQL.
+
+Specifics of the syntax of PostgreSQL can be found in this [link](https://www.postgresql.org/docs/current/sql-syntax.html).
+
+As a quick runover, to start using PostgreSQL in a Python program, you must first setup the psycopg2 module in a Python file. 
```

Comment:
Add a heading for the section where you provide instructions on how to use PostgreSQL with psycopg2